optimum/intel/openvino/configuration.py

```diff
@@ -46,18 +46,47 @@
         "dataset": "wikitext2",
         "awq": True,
     },
-    "meta-llama/Llama-2-7b": {"bits": 4, "sym": True, "group_size": 128, "ratio": 0.7},
+    "meta-llama/Llama-2-7b": {"bits": 4, "sym": True, "group_size": 128, "ratio": 0.6},
     "meta-llama/Llama-2-7b-chat": {"bits": 4, "sym": True, "group_size": 128, "ratio": 0.8},
     "meta-llama/Llama-2-13b-chat": {"bits": 4, "sym": True, "group_size": 64, "ratio": 0.8},
-    "stabilityai/stablelm-3b-4e1t": {"bits": 4, "sym": True, "group_size": 64, "ratio": 0.8},
-    "stablelm-epoch-3b-preview": {"bits": 4, "sym": True, "group_size": 64, "ratio": 0.8},
-    "stabilityai/stablelm-zephyr-3b": {"bits": 4, "sym": False, "group_size": 128, "ratio": 1.0},
+    "stabilityai/stablelm-3b-4e1t": {
+        "bits": 4,
+        "sym": True,
+        "group_size": 64,
+        "ratio": 0.8,
+        "dataset": "wikitext2",
```

Comment:
Is there a chance that using a specific dataset in the default configs causes better accuracy when evaluating on wikitext, but worse for actual use-cases?

Reply:
I think that this is possible. 
But this is only alignment with the default state from here [openvino.genai](https://github.com/openvinotoolkit/openvino.genai/blob/master/llm_bench/python/utils/nncf_utils.py), configurations aren't new.
cc @AlexKoff88

Reply:
Let's skip it here. This task is not to find better configs but rather align optimum-intel configs with ones are currently used in the OV validation which will also switch to optimum-intel. Without it, we can't switch.